include-refactoring/xsimd/arch/xsimd_avx512f.hpp

```diff
@@ -280,11 +280,11 @@ namespace xsimd {
     }
 
     template<class A> batch<float, A> bitwise_not(batch<float, A> const& self, requires<avx512f>) {
-      return _mm256_xor_ps(self, _mm256_castsi256_ps(_mm256_set1_epi32(-1)));
+      return _mm512_xor_ps(self, _mm512_castsi512_ps(_mm512_set1_epi32(-1)));
```

Comment:
wow, did you find that one when adding new tests?

Reply:
Yes, I added the `bitwise_not` test.